app/components/tag.py

```diff
@@ -0,0 +1,33 @@
+from enum import Enum
+
+
+class TagColor(Enum):
+    GREY = "govuk-tag--grey"
```

Comment:
what do you think about naming these instead of having them as colours?
e.g INACTIVE = "govuk-tag--grey"
and calling the class TagType or something like that? It would save devs having to work out which colour is used for which status.